source/docs/03-metadata.xml

```diff
@@ -1382,156 +1354,308 @@
-                     <p>Within <gi scheme="MEI">surface</gi>, each <gi scheme="MEI">graphic</gi> element may specify its dimensions using the following attributes:</p>
+                  <div xml:id="physicalProperties" type="div4">
+                     <head>Physical Properties</head>
+                     <p>Even if the section <ptr target="#msdesc"/> is about how do a physical description this part is explaining elements on the medium level.</p>
                      <p>
                         <specList>
-                           <specDesc key="att.height" atts="height"/>
-                           <specDesc key="att.width" atts="width"/>
+                           <specDesc key="extent"/>
+                           <specDesc key="dimensions"/>
+                           <specDesc key="dim"/>
+                           <specDesc key="watermark"/>
+                           <specDesc key="stamp"/>
+                           <specDesc key="physMedium"/>
+                           <specDesc key="scoreFormat"/>
                         </specList>
                      </p>
-                     <p>The values of those attributes, however, specify the height and width of the digital resource, the scan of the source, and they are typically given in pixels (see <ptr target="#facsimileElements"/>). In contrast, <gi scheme="MEI">folium</gi> and <gi scheme="MEI">bifolium</gi> are supposed to provide the dimensions of the original source in cm or inches. This makes it possible to combine separate parts of a manuscripts stored in different libraries, which are scanned at different resolutions. In case of <gi scheme="MEI">bifolium</gi> elements, these dimensions apply to the <hi rend="italic">folded</hi> sheet.</p>
-                     <p>Some printed scholarly editions like the <hi rend="italic">Neue Bach-Ausgabe</hi> provide very detailed information about the sizes and binding of individual leaves of a manuscript; with <gi scheme="MEI">foliaDesc</gi> and its children it is possible to encode that information in processable ways, even without providing digitizations of the sources via <gi scheme="MEI">surface</gi>.</p>
+                     <p>Extention and Dimension</p>
+                     <p>Watermark</p>
+                     <p>Stamp</p>
+                     <p>physMedium</p>
+                     <p>The <gi scheme="MEI">scoreFormat</gi> element is a form of classification. This element is part of <gi scheme="MEI">physDesc</gi> because within the MARC21 standard, the format of the music (score, piano score, etc.) is defined as a physical property.</p>
+                        <p>For describing the <gi scheme="MEI">scoreFormat</gi> the value list of MARC21 (see “20 - Format of music (006/03)”, https://www.loc.gov/marc/bibliographic/bd008m.html) should be used. If this value list is not sufficient, however, it is recommended to design an own classification (see <gi scheme="MEI">classification</gi>).</p>
```

Comment:
The indent levels are weird here...

Reply:
Does the MARC URL have to be wrapped in a linking tag so it is a hyperlink?

Reply:
```suggestion
                        <p>For describing the <gi scheme="MEI">scoreFormat</gi> the value list of MARC21 (see “20 - Format of music (006/03)”, https://www.loc.gov/marc/bibliographic/bd008m.html) should be used. If this value list is not sufficient, however, it is recommended to design your own classification (see <gi scheme="MEI">classification</gi>).</p>
```

Reply:
Is it really recommended to design your own? If so, shouldn't we first suggest `authority` and `authURI` to at least acknowledge that you may use other authorities as well? (e.g., RDA: https://www.rdaregistry.info/termList/formatNoteMus/)

Reply:
Yes, we should open this a bit. The idea behind the "own classification" is that the needs of a project can be different to MARC or RDA. We wanted to give a hint that there is another way to classify very specific data for special purpose. I'll make a proposal.